app/src/github.com/harithj/ci-cd-2019/github/webhook/handlers.go

```diff
@@ -0,0 +1,35 @@
+package webhook
+
+import (
+  "fmt"
+  "net/http"
+  "encoding/json"
+  "io/ioutil"
+
+  "github.com/harithj/ci-cd-2019/github/payload"
+  "github.com/harithj/ci-cd-2019/pipeline/pipeline"
+)
+
+func Payloadfunc(w http.ResponseWriter, r *http.Request) {
+  var err error
+  var actualPayload payload.ActualPayload
+  var payloadBytes []byte
+
+  payloadBytes, err = ioutil.ReadAll(r.Body)
+  if err != nil {
+    fmt.Println(err)
+  }
+
+  fmt.Println(payloadBytes)
+
+  err = json.Unmarshal(payloadBytes, &actualPayload)
+  if err != nil {
+    fmt.Println(err)
```

Comment:
It would make sense to stop execution at this point (return an error response) instead of continuing execution